internal/impl/config.go

```diff
@@ -25,7 +25,10 @@ type Config struct {
 
 	// Shell configures the devbox shell environment.
 	Shell struct {
+		// Env contains environment variables that will be set in shell
```

Comment:
Add each comment above the field it describes

Reply:
I'll change it. The reason for this comment re-organizing was that go-lint was automatically format this and make it so that `json:"xyz,omitempty"` was misaligned see comparison below:
```go
Shell struct {
		// Env contains environment variables that will be set in shell
		// InitHook contains commands that will run at shell startup.
		// Scripts contain commands that can be executed via devbox run
		Env      map[string]string             `json:"env,omitempty"`
		InitHook shellcmd.Commands             `json:"init_hook,omitempty"`
		Scripts  map[string]*shellcmd.Commands `json:"scripts,omitempty"`
	} `json:"shell,omitempty"`
```
vs
```go
Shell struct {
		// Env contains environment variables that will be set in shell
		Env map[string]string `json:"env,omitempty"`
		// InitHook contains commands that will run at shell startup.
		InitHook shellcmd.Commands `json:"init_hook,omitempty"`
		// Scripts contain commands that can be executed via devbox run
		Scripts map[string]*shellcmd.Commands `json:"scripts,omitempty"`
	} `json:"shell,omitempty"`
```